README.md

```diff
@@ -103,21 +124,50 @@ mkdir -p $HOME/.local/bin && cp lite-xl $HOME/.local/bin/
 mkdir -p $HOME/.local/share/lite-xl && cp -r data/* $HOME/.local/share/lite-xl/
 ```
 
+#### Add Lite XL to PATH
+
 If `$HOME/.local/bin` is not in PATH:
 
 ```sh
 echo -e 'export PATH=$PATH:$HOME/.local/bin' >> $HOME/.bashrc
 ```
 
-To get the icon to show up in app launcher:
+> **Note**
+> Some systems might not load `.bashrc` when logging in.
```

Comment:
Should we add the new `~/.config/environment.d/` way too? https://wiki.archlinux.org/title/environment_variables#Per_Wayland_session